app/views/candidate_interface/gcse/statement_comparability/_form.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= f.govuk_radio_buttons_fieldset :enic_details, legend: { text: t('application_form.gcse.enic_statement.enter_enic', subject: @subject), size: 'l' } do %>
```

Comment:
It would be easier to grok this if we set a local variable `scope` and passed it to the `translate` method

https://guides.rubyonrails.org/i18n.html#basic-lookup-scopes-and-nested-keys